ksp_plugin/interface.hpp

```diff
@@ -155,7 +156,8 @@ KeplerianElements ToKeplerianElements(
 QP ToQP(DegreesOfFreedom<World> const& dof);
 QP ToQP(RelativeDegreesOfFreedom<AliceSun> const& relative_dof);
 
-Status ToStatus(base::Status const& status);
+Status const* ToStatus(base::Status const& status);
+Status const* ToStatus(base::Error error, std::string const& message);
```

Comment:
> return std::unique_ptr
Hm, that does not work since Status is not RAII because of its C string.

Document that ownership is returned, and call it `MakeStatus` or `NewStatus`.

Reply:
`ToNewStatus`.